_rules/inline-link-in-paragraph-is-distinguishable-be4d0c.md

```diff
@@ -0,0 +1,488 @@
+- [F73: Failure of Success Criterion 1.4.1 due to creating links that are not visually evident without color vision](https://www.w3.org/WAI/WCAG21/Techniques/failures/F73)
+- [C15: Using CSS to change the presentation of a user interface component when it receives focus](https://www.w3.org/WAI/WCAG21/Techniques/css/C15)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This link, with a **different hue** from the other text in the same line, uses a style similar to the default styling of links which underlines them in most browsers, making it a distinguishable style.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: underline;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 2
+
+This link, with a **different hue** from the other text in the same line, uses a style similar to the default styling of links (underline) when it is focused or hovered.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		text-decoration: none;
+		color: blue;
+		background-color: white;
+	}
+	a:hover,
+	a:focus {
+		text-decoration: underline;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 3
+
+This link, with a background with a **different hue** from the other text in the same line, uses a style similar to the default styling of links which underlines them in most browsers, making it a distinguishable style.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: black;
+		background-color: #indianred;
+		text-decoration: underline;
+	}
+</style>
+<p>Read about WAI on the <a href="http://w3.org/WAI">WAI webpage</a>.</p>
+```
+
+#### Passed Example 4
+
+This element with a [semantic role][] that inherits from link, with a **different hue** from the other text in the same line, uses a style similar to the default styling of links which underlines them in most browsers, making it a distinguishable style.
+
+```html
+<style>
+	p {
+		color: black;
+		background-color: white;
+	}
+	a {
+		color: blue;
+		background-color: white;
+		text-decoration: underline;
+	}
+</style>
+<p>This is indicated in the cost of a <a href="#bcc0f155" role="doc-glossref">credit default swap</a>.</p>
+<h1>Glossary</h1>
+<p id="bcc0f155">
+	A credit default swap (CDS) is a financial derivative or contract that allows an investor to "swap" or offset his or
+	her credit risk with that of another investor.
+</p>
+```
+
+#### Passed Example 5
+
+This link, with a **different hue** from the other text in the same line, has a distinguishable color because the color contrast is 4.98.
```

Comment:
Contrast is only accurate to a single digit and needs to be expressed as a ratio.

```suggestion
This link, with a **different hue** from the other text in the same line, has a distinguishable color because the color contrast is 5.0:1.
```